README.md

```diff
@@ -1,3 +1,44 @@
 # ctypehub
 
 The source code of https://ctypehub.galaniprojects.de
+
+### Prerequisites
+
+Make sure you installed these required software before running this project:
+
+- [Node.js](https://nodejs.org/en/download/prebuilt-installer)
+- [pnpm](https://pnpm.io/installation#using-corepack)
+- [Docker](https://docs.docker.com/engine/install/)
+
+
+## Start the app locally:
+
+After having [cloned the repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) on your local machine and defining all environment variables according to the `.env.example`, to start the app locally follow this steps.
```

Comment:
Is it clear from this line how to define environment variables?

Reply:
I mean, the `.env.example` does not explain the variables that much. 

It is indeed not ideal.

Reply:
It's better as before though.